src/css/custom.css

```diff
@@ -68,9 +68,34 @@ h2,
 h3,
 h4,
 h5 {
-  font-weight: 500;
+  font-weight: 700;
 }
 
+h1 {
```

Comment:
Great, you do this! Thank you very much! Just a heads up, maybe we can include this in a separate PR to have it merged quicker, the topic is as well not related to the topic of this article. But great, thanks!